.travis.yml

```diff
@@ -22,15 +25,19 @@ before_cache:
 
 matrix:
   include:
-    - env: CABALVER=head GHCVER=7.8.4
+    - env: CABALVER=head GHCVER=7.8.4 ELASTICSEARCH=1.7
```

Comment:
@arybczak Yeah, it doesn't seem to be a problem even though it is not "officially" supported by Bloodhound.